### PR TITLE
Fix legacy-style data dictionary links

### PR DESCRIPTION
**Pull request by austin-schaefer**
Removing remaining legacy-style data dictionary links from the site. Checked a bunch of links in Gatsby Cloud and all seem fine.

### CODE REVIEW

File: src/content/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui/infrastructure-hosts-page.mdx [hunk 1]
```diff
@@ -267,7 +267,7 @@ Here are some of the things you can do from the **Network** tab:
...
+  By default, the infrastructure agent doesn't send [data about the operating system's processes](https://docs.newrelic.com/attribute-dictionary/?event=ProcessSample) unless you use guided install. To enable the sending of process data set [`enable_process_metrics`](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/) to `true`. To fine-tune which processes you want to monitor, configure [`include_matching_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#include-matching-metrics).
```
Comment (stumblefiend) [inline src/content/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui/infrastructure-hosts-page.mdx:270]: how come this isn't a relative URL?

File: src/content/docs/infrastructure/infrastructure-troubleshooting/troubleshoot-infrastructure/no-data-appears-infrastructure.mdx [hunk 2]
```diff
@@ -29,7 +29,7 @@ Data should appear in the Infrastructure monitoring UI within a few minutes for
...
+  By default, the infrastructure agent doesn't send data about the operating system's processes. To enable the sending of process data set [`enable_process_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#enable-process-metrics) to `true`. To fine-tune which processes you want to monitor, configure [`include_matching_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#include-matching-metrics).
```
Comment (stumblefiend) [inline src/content/docs/infrastructure/infrastructure-troubleshooting/troubleshoot-infrastructure/no-data-appears-infrastructure.mdx:32]: how come these aren't relative URLs?

File: src/content/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings.mdx [hunk 3]
```diff
@@ -2564,7 +2564,7 @@ If the agent is running in a cloud instance, the agent will try to detect the cl
 
     </Callout>
 
-    Enables the sending of [process metrics](https://docs.newrelic.com/attribute-dictionary?attribute_name=&events_tids%5B%5D=8412) to New Relic.
+    Enables the sending of [process metrics](https://docs.newrelic.com/attribute-dictionary/?event=ProcessSample) to New Relic.
```
Comment (stumblefiend) [inline src/content/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings.mdx:2567]: relative URL?

File: src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-instrumentation-infrastructure-monitoring.mdx [hunk 4]
```diff
@@ -49,20 +49,20 @@ For more information, see [Query your data](/docs/using-new-relic/data/understan
...
-To see all `ContainerSample` attributes, use our [data dictionary](/attribute-dictionary?attribute_name=&events_tids%5B0%5D=10181). Attributes include:
+To see all `ContainerSample` attributes, use our [data dictionary](/attribute-dictionary/?event=ContainerSample). Attributes include:
 
 * General metadata (like `containerId`, `name`, and `image`)
 * CPU metrics (like `cpuUsedCores`, `cpuPercent`, and `cpuThrottleTimeMs`)
 * Memory metrics (like `memoryUsageBytes`, `memoryCacheBytes`, and `memoryResidentSizeBytes`)
 * Network metrics (like `networkRxBytes`, `networkRxDropped`, and `networkTxBytes`)
 
-Docker metrics are also attached to the [`ProcessSample`](https://docs.newrelic.com/attribute-dictionary?attribute_name=&events_tids%5B%5D=8412) event.
+Docker metrics are also attached to the [`ProcessSample`](https://docs.newrelic.com/attribute-dictionary/?event=ProcessSample) event.
```
Comment (stumblefiend) [inline src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-instrumentation-infrastructure-monitoring.mdx:65]: another potential relative URL?

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/alert-infrastructure-processes.mdx [hunk 5]
```diff
@@ -19,7 +19,7 @@ Use New Relic infrastructure's **Process running** [alert condition](/docs/infra
...
+  By default, the infrastructure agent doesn't send [data about the operating system's processes](https://docs.newrelic.com/attribute-dictionary/?event=ProcessSample). To enable the sending of process data set [`enable_process_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#enable-process-metrics) to `true`. To fine-tune which processes you want to monitor, configure [`include_matching_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#include-matching-metrics).
```
Comment (stumblefiend) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/alert-infrastructure-processes.mdx:22]: relative?

File: src/content/docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx [hunk 6]
```diff
@@ -116,7 +116,7 @@ The New Relic Kubernetes integration has the following requirements:
 * A New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
 * Linux distribution [compatible with New Relic infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
 * [`kube-state-metrics` version 1.9.8](https://github.com/kubernetes/kube-state-metrics) running on the cluster.
-* When using CRI-O as the container runtime, the [processes](https://docs.newrelic.com/attribute-dictionary?attribute_name=&events_tids%5B%5D=8412) inside containers are not reported. Performance data is collected at the [container](https://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data#container-data) level.
+* When using CRI-O as the container runtime, the [processes](https://docs.newrelic.com/attribute-dictionary/?event=ProcessSample) inside containers are not reported. Performance data is collected at the [container](https://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data#container-data) level.
```
Comment (stumblefiend) [inline src/content/docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx:119]: relative?

File: src/content/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-feature-end-life-announcements-july-2020.mdx [hunk 7]
```diff
@@ -250,7 +250,7 @@ Documentation:
 
 * [Explorers Hub post](https://discuss.newrelic.com/t/replicate-the-connected-agents-report-in-insight/77575)
 * [NRQL query tools](/docs/query-your-data/nrql-new-relic-query-language/query-tools)
-* [NrDailyUsage data definition](https://docs.newrelic.com/attribute-dictionary?attribute_name=&events_tids%5B%5D=8891)
+* [NrDailyUsage data definition](https://docs.newrelic.com/attribute-dictionary/?event=NrDailyUsage)
```
Comment (stumblefiend) [inline src/content/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-feature-end-life-announcements-july-2020.mdx:253]: relative?

File: src/content/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui/infrastructure-hosts-page.mdx (hunk 1 above)
Comment (austin-schaefer) [reply src/content/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui/infrastructure-hosts-page.mdx]: Good catch. Looks like they were absolute originally and I missed that in the sweep. Just pushed up a fresh commit to fix these should-be-relative links.

File: src/content/docs/agents/ruby-agent/attributes/enable-disable-attributes-ruby.mdx [hunk 8]
```diff
@@ -467,7 +467,7 @@ Use the following properties to enable or disable attributes:
 
     Prefix of attributes to exclude from error collection. Allows `*` as wildcard at end.
 
-    To prevent [error.message](https://docs.newrelic.com/attribute-dictionary?attribute_name=error.message) collection, instead use the [strip_exception_messages](https://docs.newrelic.com/docs/agents/ruby-agent/configuration/ruby-agent-configuration#strip_exception_messages-enabled) configuration.
+    To prevent [error.message](/attribute-dictionary?attribute_name=error.message) collection, instead use the [strip_exception_messages](https://docs.newrelic.com/docs/agents/ruby-agent/configuration/ruby-agent-configuration#strip_exception_messages-enabled) configuration.
```
Comment (stumblefiend) [inline src/content/docs/agents/ruby-agent/attributes/enable-disable-attributes-ruby.mdx:470]: another relative link here

File: src/content/docs/browser/new-relic-browser/page-load-timing-resources/pageviewtiming-async-or-dynamic-page-details.mdx [hunk 9]
```diff
@@ -79,7 +79,7 @@ The `BrowserInteraction` and `PageView` events end their reporting when they rec
...
+        This new metric, available with [agent version 1163](https://docs.newrelic.com/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1163), [`largestContentfulPaint`](/attribute-dictionary/pageviewtiming/largestcontentfulpaint), reports the render time of the largest content element visible in the viewport. Google's research found looking at when the largest element was rendered was a more accurate way to measure when the main content of a page is loaded and useful. Details on this metric, including limitations and considerations can be found in the [w3c draft](https://wicg.github.io/largest-contentful-paint/).
```
Comment (stumblefiend) [inline src/content/docs/browser/new-relic-browser/page-load-timing-resources/pageviewtiming-async-or-dynamic-page-details.mdx:82]: needs relative link

File: src/content/docs/infrastructure/infrastructure-troubleshooting/troubleshoot-infrastructure/no-data-appears-infrastructure.mdx (hunk 2 above)
Comment (stumblefiend) [inline src/content/docs/infrastructure/infrastructure-troubleshooting/troubleshoot-infrastructure/no-data-appears-infrastructure.mdx:32]: needs relative links

File: src/content/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui/infrastructure-hosts-page.mdx [hunk 10]
```diff
@@ -267,7 +267,7 @@ Here are some of the things you can do from the **Network** tab:
...
+  By default, the infrastructure agent doesn't send [data about the operating system's processes](/attribute-dictionary/?event=ProcessSample) unless you use guided install. To enable the sending of process data set [`enable_process_metrics`](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/) to `true`. To fine-tune which processes you want to monitor, configure [`include_matching_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#include-matching-metrics).
```
Comment (stumblefiend) [inline src/content/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui/infrastructure-hosts-page.mdx:270]: relative link

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/alert-infrastructure-processes.mdx [hunk 11]
```diff
@@ -19,7 +19,7 @@ Use New Relic infrastructure's **Process running** [alert condition](/docs/infra
...
+  By default, the infrastructure agent doesn't send [data about the operating system's processes](/attribute-dictionary/?event=ProcessSample). To enable the sending of process data set [`enable_process_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#enable-process-metrics) to `true`. To fine-tune which processes you want to monitor, configure [`include_matching_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#include-matching-metrics).
```
Comment (stumblefiend) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/alert-infrastructure-processes.mdx:22]: needs relative links

File: src/content/docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx [hunk 12]
```diff
@@ -116,7 +116,7 @@ The New Relic Kubernetes integration has the following requirements:
 * A New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
 * Linux distribution [compatible with New Relic infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
 * [`kube-state-metrics` version 1.9.8](https://github.com/kubernetes/kube-state-metrics) running on the cluster.
-* When using CRI-O as the container runtime, the [processes](https://docs.newrelic.com/attribute-dictionary?attribute_name=&events_tids%5B%5D=8412) inside containers are not reported. Performance data is collected at the [container](https://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data#container-data) level.
+* When using CRI-O as the container runtime, the [processes](/attribute-dictionary/?event=ProcessSample) inside containers are not reported. Performance data is collected at the [container](https://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data#container-data) level.
```
Comment (stumblefiend) [inline src/content/docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx:119]: I know some of these links weren't part of your sweep, but should they be made relative too?

**Review by stumblefiend — CHANGES_REQUESTED** [review]

**Review by stumblefiend — APPROVED** [review]

File: src/content/docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx (hunk 12 above)
Comment (austin-schaefer) [reply src/content/docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx:119]: I'm inclined to not block on making the other links relative, there are around ~2000 absolute links on the site that probably "should" be relative links and I feel like it's a pretty bottomless rabbithole.

**Comment by austin-schaefer** [comment]
Ended up merging w/o fixing all relative links, there are about ~2000 absolute links on the docs site that "should" be relative and I'm not inclined to go too far down that rabbithole.

**Pull request merged**